C-Plus-Plus/graphs/Kosaraju_Algorithm.cpp

```diff
@@ -0,0 +1,88 @@
+#include<bits/stdc++.h>
```

Comment:
@devanshiv123 add a comment `Implementation of Kosaraju's Algorithm to find out the strongly connected
   components (SCCs) in a graph.`